entity-service/src/integrationTest/java/org/hypertrace/entity/service/service/EntityQueryServiceTest.java

```diff
@@ -82,6 +90,7 @@
   private static final String SERVICE_ID = generateRandomUUID();
 
   private static Map<String, String> apiAttributesMap;
+  private static Map<String, String> headers = Map.of("x-tenant-id", TENANT_ID);
```

Comment:
statis final variables are upper cased 

```suggestion
  private static final Map<String, String> HEADERS = Map.of("x-tenant-id", TENANT_ID);
```